vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -691,15 +691,10 @@ func (e *StandardTxExecutor) RegisterSubnetValidatorTx(tx *txs.RegisterSubnetVal
 	if err != nil {
 		return err
 	}
-	if warpMessage.NetworkID != e.Ctx.NetworkID {
```

Comment:
I'm assuming this was only ever added as a stopgap defensive check on the branch prior to ICM verification being added?

Reply:
After rebasing, this diff no longer exists :+1:.